src/libcore/va_list.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![unstable(feature = "c_variadic",
+            reason = "dlrobertson is still working on this",
+            issue = "27745")]
+
+//! Implementation of a `va_list`
+
+#[cfg(not(stage0))]
+#[lang = "va_list"]
+/// A wrapper for a `va_list`
+//#[derive(Debug)]
+#[repr(C)]
+#[unstable(feature = "c_variadic",
+           reason = "dlrobertson is still working on this",
+           issue = "27745")]
+#[derive(Debug)]
+pub struct VaList;
```

Comment:
The original RFC indicates that this should be implemented as `core::intrinsics::VaList`, but since everything in `intrinsics` is a `"rust-intrinsic"` I implemented it as `core::va_list::VaList`. Was there a reason this was going to be implemented under `intrinsics`?